Quadriga/src/main/webapp/WEB-INF/views/workspace/workspacedetails.jsp

```diff
@@ -589,8 +599,106 @@
 										}
 									});
 				}
-			</script> 
 
+                function confirmArchive(isArchive) {
+                    isArchive = !!isArchive;
+                    var txt = isArchive ? 'Archive' : 'Unarchive';
+                    var pos = [ $(window).width() / 4, 50 ];
+                    var url = '${pageContext.servletContext.contextPath}/auth/workbench/${myprojectid}';
+                    var path = isArchive ? '/archiveworkspace' : '/unarchiveworkspace';
+                    var title = isArchive ? 'Archive Workspace' : 'Unarchive Workspace';
+                    path += '/${workspaceid}';
+                    console.log(url + path)
+                    $('#dialog-confirm')
+                            .html('Are you sure you want to ' + txt + ' this workspace?')
+                            .dialog({
+                                resizable : false,
+                                modal : true,
+                                title : title,
+                                height : 180,
+                                width : 650,
+                                position : pos,
+                                buttons : {
+                                    "Yes" : function() {
+                                        $(this).dialog('close');
+                                        location.href = url + path;
+                                        return false;
+                                    },
+                                    "No" : function() {
+                                        $(this).dialog('close');
+                                        return false;
+                                    }
+                                }
+                            });
+                }
+			</script> <c:choose>

```

Comment:
The choose is used for other check for some other functionality. I formatted the code.